### PR TITLE
Feature/12219 testprocess family members/12340 create family member consent screen

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Added new family test consent screen to enter name of family member. Included the screen to the flow into coordinator and added some simple unit tests.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12340

## Screenshots
Simulator Screen Shot - iPhone 8 - 2022-03-23 at 17 41 45
Simulator Screen Shot - iPhone 8 - 2022-03-23 at 17 42 23
Simulator Screen Shot - iPhone 8 - 2022-03-23 at 17 51 03
Simulator Screen Shot - iPhone 8 - 2022-03-23 at 17 51 07
Simulator Screen Shot - iPhone 8 - 2022-03-23 at 17 51 10

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -3050,6 +3050,32 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
 
+/* HealthCertificate Family Consent */
+
+"HealthCertificate_FamilyMemberConsent_Title" = "Ihr Einverständnis";
+
+"HealthCertificate_FamilyMemberConsent_ImageDescription" = "Eine Familie mit mehreren Mitgliedern; im Hintergrund ein Dokument mit QR-Code darauf.";
+
+"HealthCertificate_FamilyMemberConsent_Headline" = "Testergebnis für Familienmitglieder abrufen";
+
+"HealthCertificate_FamilyMemberConsent_InputTitle" = "Name der Person, die geteste wurde:";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3061]:
```suggestion
"HealthCertificate_FamilyMemberConsent_InputTitle" = "Name der Person, die getestet wurde:";
```

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
correct typo

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/FamilyMemberConsent/FamilyNameTextFieldCell.swift [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		model = textField.text
+	}
+
+	func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+		endEditing(true)
+	}
+
+	// MARK: - Internal
+
+	@OpenCombine.Published private(set) var model: String?
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/FamilyMemberConsent/FamilyNameTextFieldCell.swift:44]:
```suggestion
	@OpenCombine.Published private(set) var name: String?
```

**Review by ArturFriesen — APPROVED** [review]

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**